nevooronni/Questioner
# #162956728 A user can post a question

#### What does this PR do?
This is a PR for a feature that allows a user to post questions to a specific meetup

#### Description of Task to be completed?
The following are the tasks to be completed
* Implement html and css for questions page
* Implement html and css for post questions modal form

#### How to manually test feature?
1. clone this repo
``https://github.com/nevooronni/Questioner.git```

2. cd in folder
```cd Questioner```

3 switch to feature branch 
```git checkout ft-post-question-162956728```

4. Open index.html in your browser

5. Click the signin button on landing page
6. Click the admin button on signin page
7. Click create meetup on meetups page
8. Click the first meetup event 'End Of Month Meetup'
9. Click the Post question button to see the modal form pop up

#### Pivotal Tracker story associated with this PR:
story id: #162956728

#### Screenshots
![screenshot from 2019-01-07 03-23-33](https://user-images.githubusercontent.com/20374625/50743744-9047db00-122c-11e9-9a0a-22adff1a2618.png)


![screenshot from 2019-01-07 03-23-11](https://user-images.githubusercontent.com/20374625/50743751-a05fba80-122c-11e9-86a2-68af689d1e8a.png)
